Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
-    highline (2.0.0)
+    highline (2.0.3)
     method_source (0.9.2)
     mustermann (1.0.3)
     pg (0.20.0)
     pry (0.12.2)
       coderay (~> 1.1.0)
       method_source (~> 0.9.0)
-    pry-byebug (3.6.0)
-      byebug (~> 10.0)
+    pry-byebug (3.7.0)
+      byebug (~> 11.0)
       pry (~> 0.10)
-    rack (2.0.6)
-    rack-protection (2.0.4)
+    rack (2.0.8)
```

Comment:
### [rack](https://rack.github.io/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.6 → 2.0.8

[_~~change-log~~, [source-code](http://github.com/rack/rack)_]

:tada: Patched vulnerabilities:

- [CVE-2019-16782](https://nvd.nist.gov/vuln/detail/CVE-2019-16782)
  **Possible information leak / session hijack vulnerability**

  
  URL: https://github.com/rack/rack/security/advisories/GHSA-hrqr-hxpp-chr3

  There's a possible information leak / session hijack vulnerability in Rack.  Attackers may be able to find and hijack sessions by using timing attacks targeting the session id. Session ids are usually stored and indexed in a database that uses some kind of scheme for speeding up lookups of that session id. By carefully measuring the amount of time it takes to look up a session, an attacker may be able to find a valid session id and hijack the session.  The session id itself may be generated randomly, but the way the session is indexed by the backing store does not use a secure comparison.  Impact:  The session id stored in a cookie is the same id that is used when querying the backing session storage engine.  Most storage mechanisms (for example a database) use some sort of indexing in order to speed up the lookup of that id.  By carefully timing requests and session lookup failures, an attacker may be able to perform a timing attack to determine an existing session id and hijack that session.






<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/rack/rack/compare/2.0.6...2.0.8).

These are the first 10 commits:
- (cb1fdb6) [Merge pull request #1201 from janko-m/make-multipart-parsing…](https://github.com/rack/rack/commit/cb1fdb600bc525258b3c34ea95f1598ee6def9c6)
- (1bf2188) [Preserve forwarded IP address for trusted proxy chains](https://github.com/rack/rack/commit/1bf218818502e820192a41c4da61aa0b0b6109af)
- (ea57610) [Merge pull request #1343 from larsxschneider/ls/forward-fix](https://github.com/rack/rack/commit/ea576109c1b9fd444e6f0e728f8db74c33786674)
- (7fb95db) [Bumping to 2.0.7 for release](https://github.com/rack/rack/commit/7fb95dbec28dc70f3cfbba0a684db0735d8ab2ca)
- (83d4bd1) [try to ensure we always have some kind of object](https://github.com/rack/rack/commit/83d4bd12c7e88455d21230bc24ec3a543654e2aa)
- (77f3aab) [remove more nils](https://github.com/rack/rack/commit/77f3aab73089abe518f62c46268b104bacd7114b)
- (bb3d486) [use session id objects](https://github.com/rack/rack/commit/bb3d486644755b2e0c7824b3910db1a83c98fcd2)
- (2b205ed) [store hashed id, send public id](https://github.com/rack/rack/commit/2b205ed5a047d9e50a13bb7a411bc48745b515ec)
- (1c7e3b2) [remove || raise and get closer to master](https://github.com/rack/rack/commit/1c7e3b259f0741c869dcfbabeb3e0670c4d3f848)
- (4e32262) [remove NullSession](https://github.com/rack/rack/commit/4e322629e0c6698c75a3fb541a42571f8543c34c)


</details>

Reply:
🚀 This is the reason for my PR.